ocf-servers/js-servers/environmental_sensor.js

```diff
@@ -18,17 +18,19 @@ var	device = require('iotivity-node')('server'),
 	sensorData =   {temperature : 0.0,
 			humidity : 0.0,
 			pressure : 0.0,
-			uvIndex : 0};
+			uvIndex : 0},
+    simData = 0.0;
 
 var noble = '';
 try {
 	noble = require('noble');
 }
 catch(e) {
 	debuglog('Require noble module: ', e.message);
```

Comment:
Actually... the way you've done it is currently how it's done in all other OCF JS servers... let's keep it as it is then. We can change that in batch mode later if we decide too...